e2e/login.spec.ts

```diff
@@ -1,31 +1,49 @@
+import fs from 'fs';
+import path from 'path';
+import { fileURLToPath } from 'url';
 import { test, expect } from '@playwright/test';
-import projects from './projects-config.json' assert { type: 'json' };
 
-// Create a loop to loop over all the projects and create a test for everyone
+type Project = {
+  env: 'prod' | 'dev' | 'test';
+  name: string;
+  url: string;
+  title: string;
+  hasCookieBanner: boolean;
+};
+
+const __filename = fileURLToPath(import.meta.url);
+const __dirname = path.dirname(__filename);
+
+const data = fs.readFileSync(path.join(__dirname, 'projects-config.json')).toString();
```

Comment:
no so stable as tests could be grouped into folders in future, why not to use 'import projects from './projects-config.json' assert { type: 'json' }; '?